src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -1044,6 +1058,46 @@ void MinimalScene::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
       renderWindow->SetCameraPose(pose);
     }
 
+    elem = _pluginElem->FirstChildElement("camera_clip");
+    if (nullptr != elem && !elem->NoChildren())
+    {
+      auto child = elem->FirstChildElement("near");
+      if (nullptr != child && nullptr != child->GetText())
+      {
+        double near;
+        std::stringstream nearStr;
+        nearStr << std::string(child->GetText());
+        nearStr >> near;
+        if (nearStr.fail())
+        {
+          ignerr << "Unable to set <near> to '" << nearStr.str()
+                 << "' using default near clip distance" << std::endl;
+        }
+        else
+        {
+          renderWindow->SetCameraNearClip(near);
+        }
+      }
+
+      child = elem->FirstChildElement("far");
+      if (nullptr != child && nullptr != child->GetText())
+      {
+        double far;
```

Comment:
recommend renaming this variable

* https://stackoverflow.com/questions/118774/is-there-a-clean-way-to-prevent-windows-h-from-creating-a-near-far-macro

Reply:
cb964f681a7e440bf76163d893f5cbef0294522d